plugin/clang.vim

```diff
@@ -877,9 +895,34 @@ func! s:ClangExecute(root, clang_options, line, col)
   let l:command .= ' 1>'.l:tmps[0].' 2>'.l:tmps[1]
   let l:res = [[], []]
   if has("nvim")
-    let l:argv = ['sh', '-c', l:command]
-    call s:PDebug("s:ClangExecute::job.argv", l:argv, 2)
-    call jobstart(l:argv, {'fstdout': l:tmps[0], 'fstderr': l:tmps[1], 'on_exit': function('ClangExecuteNeoJobHandler')})
+    let l:command = printf('%s -cc1 -x c++ -fsyntax-only -code-completion-macros -code-completion-at -:%d:%d %s',
+                      \ g:clang_exec, a:line, a:col, a:clang_options)
+    call s:PDebug("s:ClangExecute::cmd", l:command, 2)
+
+    " try to force stop last job which doesn't exit.
+    if exists('b:jobid')
```

Comment:
Please use `b:clang_execute_neojob_id` or something like it that has a prefix.
And then add comment at the beginning of the function.
```
" Buffer var b:clang_... 
```